src/Task/ApiGen/ApiGen.php

```diff
@@ -30,6 +31,7 @@ class ApiGen extends BaseTask implements CommandInterface
      * @var string
      */
     protected $command;
+    protected $operation = ' generate';
```

Comment:
Don't include the space in `$operation`; this is harder to read, and inconsistencies can lead to errors. Instead, ensure that appropriate spacing is added when command is composed, and keep the various parts trimmed.